pkgs/applications/misc/worker/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchurl }:
+let
+pkgs = import <nixos> {};
+in
```

Comment:
Please remove the line `pkgs = import <nixos> {};` and import `libX11` in the first line:

```
{ stdenv, fetchurl, libX11 }:
```